Header header
drrobot_h20_arm_player/arm_cmd[] left_trajectory
drrobot_h20_arm_player/arm_cmd[] right_trajectory
